### PR TITLE
ready for review

### PR DESCRIPTION
**Pull request by rOluochKe**
data model
validation
association

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema.define(version: 2019_12_17_190720) do
+
+  create_table "comments", force: :cascade do |t|
+    t.string "body"
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
+  end
```
Comment (maelfosso) [inline db/schema.rb]: Please, where is the 
- post_id 
- user_id
??

Comment (rOluochKe) [reply db/schema.rb]: all the issues are solved

Comment (rOluochKe) [reply db/schema.rb]: all issues are solved

File: app/models/comment.rb [hunk 2]
```diff
@@ -3,5 +3,5 @@
 class Comment < ApplicationRecord
   validates :body, presence: true, length: { minimum: 6, maximum: 50 }
 
-  belongs_to :user
+  belongs_to :post
```
Comment (samgaco) [inline app/models/comment.rb]: - [ ] the comments are to be made in a post by a user, so the relation belongs_to  :user  was necessary as well, please add that back

Comment (rOluochKe) [reply app/models/comment.rb]: added,
thank you

**Pull request merged**